.github/workflows/update_readme.yml

```diff
@@ -0,0 +1,36 @@
+# -----------------------------------------------------------------------------
+#   - invoked on PR to main branch
+# -----------------------------------------------------------------------------
+name: update README
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  update-readme:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.11
+      - name: Update README
+        run: |
+          pip install packaging
+          python update_readme.py
+      - name: Commit and push changes
+        run: |
+          git config --local user.email "action@github.com"
```

Comment:
Does this make the committer the author of the pull request?

Reply:
iirc (I squashed the commits so can't know for sure) the commits came in clearly from github

Reply:
I take it back - the commits still come from the user (somehow)